src/main/scala/io/smartdatalake/util/hdfs/Partition.scala

```diff
@@ -54,6 +54,21 @@ private[smartdatalake] case class PartitionValues(elements: Map[String, Any]) {
 private[smartdatalake] object PartitionValues {
   val singleColFormat = "<partitionColName>=<partitionValue>[,<partitionValue>,...]"
   val multiColFormat = "<partitionColName1>=<partitionValue>,<partitionColName2>=<partitionValue>[;(<partitionColName1>=<partitionValue>,<partitionColName2>=<partitionValue>;...]"
+  def getOrdering(partitions: Seq[String]): Ordering[PartitionValues] = new Ordering[PartitionValues] {
```

Comment:
What is the purpose of this method?
What is meant by partitions and PartitionValues?
I was not able to find an explanation in scala doc.

Reply:
@kaelte: I added scaladoc and tests for it. Can you please check again?